src/linker/Linker/MethodDefinitionExtensions.cs

```diff
@@ -71,6 +71,9 @@ public static bool IsStaticConstructor (this MethodDefinition method)
 			return method.IsConstructor && method.IsStatic;
 		}
 
+#if !FEATURE_ILLINK
```

Comment:
What's the rational for keeping it around if it's wrong?

Reply:
Monolinker build break.